code/go/internal/validator/semantic/validate_fields_limits.go

```diff
@@ -0,0 +1,65 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package semantic
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	ve "github.com/elastic/package-spec/code/go/internal/errors"
+	"github.com/elastic/package-spec/code/go/internal/fspath"
+)
+
+const maxFieldsPerDataStream = 1024
```

Comment:
Not sure if moving this too to somewhere in the spec :thinking:

Reply:
I would consider every number/count/etc. as a constant value, so - in the spec :)

Reply:
Moved https://github.com/elastic/package-spec/pull/278/commits/9453025cf021e269e8aa36a1fd40af7d73a35b5f